README.md

```diff
@@ -17,6 +17,14 @@ juju deploy vault-k8s -n 5 --trust
 
 We recommend deploying Vault with an odd number of units.
 
+### Integrate with Ingress
+
+
+```bash
+juju deploy traefik-k8s --trust --config external_hostname=<your hostname> --config routing_mode=subdomain
```

Comment:
Is subdomain needed? Why not use the default option

Reply:
Subdomain is not necessary. Removed.